pkg/proxy/iptables/proxier_test.go

```diff
@@ -1802,49 +1796,47 @@ COMMIT
 -A KUBE-POSTROUTING -m comment --comment "kubernetes service traffic requiring SNAT" -j MASQUERADE
 -A KUBE-MARK-MASQ -j MARK --or-mark 0x4000
 -A KUBE-SERVICES -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 172.30.0.41 --dport 80 -j KUBE-SVC-XPGD46QRK7WJZT7O
--A KUBE-SVC-XPGD46QRK7WJZT7O -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 172.30.0.41 --dport 80 ! -s 10.0.0.0/8 -j KUBE-MARK-MASQ
+-A KUBE-SVC-XPGD46QRK7WJZT7O -d 172.30.0.41 ! -s 10.0.0.0/8 -j KUBE-MARK-MASQ
```

Comment:
These comments had previously been quite helpful when debugging, any reason for removing them?

Reply:
The testcase diff makes it look like I just ripped out the comment, but it's actually a different rule now, generated in a different part of the code, and in that part of the code, we don't already have access to that comment string, so I just left it without a comment.

I feel like we don't need a comment on _every_ rule... As long as there is _some_ rule in the `KUBE-SVC-XPGD46QRK7WJZT7O` chain with the service name in it (which there is), then it seems like that serves the debugging need.

(And the comments cause some performance issues at scale; there's already code to omit some of them in some cases.)

But anyway, yes, we could add the comment back

Reply:
I just learned that rules can exist that JUST have comments.  Should we consider doing less per-rule comments and more per-chain ?

Reply:
(as a followup!!)